app/user/schemas/create_schemas.py

```diff
@@ -0,0 +1,30 @@
+from pydantic import BaseModel, EmailStr, Field
+
+
+class UserCreate(BaseModel):
+    """The User creation model"""
+
+    profile_picture_url: str = Field(
```

Comment:
No need to add this whole creating, if you check the design when creating user there isn't a field to upload image

Reply:
There isn't during sign up but it's in the app

Reply:
UserCreate is used for signup na?